src/testdir/test_ins_complete.vim

```diff
@@ -142,7 +142,15 @@ func Test_completefunc_args()
   delfunc CompleteFunc
 endfunc
 
-func s:CompleteDone_CompleteFuncDict( findstart, base )
+function! s:CompleteDone_CompleteFuncNone( findstart, base )
```

Comment:
Patch 8.1.0711 replaced `function!` into `func` (without exclamation
mark) in tests. So we should stay consistent with the new style now.